ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsFlatStreamReader.scala

```diff
@@ -161,6 +161,7 @@ class TransactionsFlatStreamReader(
       ids.async
         .addAttributes(
           Attributes.inputBuffer(
+            // TODO: Consider use the nearest greater or equal power of two instead to prevent stream creation failures
```

Comment:
Just checking, but I guess you're going to address this as part of the same epic